crypto/stack/stack.c

```diff
@@ -149,7 +149,7 @@ void *OPENSSL_sk_delete_ptr(OPENSSL_STACK *st, const void *p)
 
 void *OPENSSL_sk_delete(OPENSSL_STACK *st, int loc)
```

Comment:
But it's passed to us as a `const void *` so we should return it as a `const void *` surely? If the application wants to do something different to it, then the application should cast it.